kasa/cli/discover.py

```diff
@@ -68,6 +66,57 @@ async def print_discovered(dev: Device):
                 discovered[dev.host] = dev.internal_state
             echo()
 
+    await _discover(ctx, print_discovered, print_unsupported)
+
+    echo(f"Found {len(discovered)} devices")
+    if unsupported:
+        echo(f"Found {len(unsupported)} unsupported devices")
+    if auth_failed:
+        echo(f"Found {len(auth_failed)} devices that failed to authenticate")
+
+    return discovered
+
+
+@discover.command()
+@click.pass_context
+async def list(ctx):
+    """Discover devices in the network using udp broadcasts."""
```

Comment:
Updated the docstring.

This returns a single table which is quite useful to avoid searching back through all the standard detailed responses if not needed.